mage_ai/io/bigquery.py

```diff
@@ -99,7 +99,11 @@ def export(
         self.client.load_table_from_dataframe(df, table_id, job_config=config).result()
 
     @classmethod
-    def with_credentials_file(cls, path_to_credentials: str, **kwargs):
+    def with_config(cls, config: Mapping[str, Any]) -> 'BigQuery':
+        return cls(**config['BigQuery'])
```

Comment:
Let's define those config keys as constants and raise exception with good error message when config is invalid.

Reply:
Moved config keys into an enum `IOConfigKeys`.